changeset:   167911:33776df74896
tag:         tip
user:        Solomon Chiu <schiu@mozilla.com>
date:        Wed Feb 12 01:50:33 2014 +0800
summary:     Rename GonkBufferQueue/GonkConsumerBase to JB version

diff --git a/widget/gonk/nativewindow/GonkBufferQueue.cpp b/widget/gonk/nativewindow/GonkBufferQueueJB.cpp
rename from widget/gonk/nativewindow/GonkBufferQueue.cpp
rename to widget/gonk/nativewindow/GonkBufferQueueJB.cpp
diff --git a/widget/gonk/nativewindow/GonkBufferQueue.h b/widget/gonk/nativewindow/GonkBufferQueueJB.h
rename from widget/gonk/nativewindow/GonkBufferQueue.h
rename to widget/gonk/nativewindow/GonkBufferQueueJB.h
diff --git a/widget/gonk/nativewindow/GonkConsumerBase.cpp b/widget/gonk/nativewindow/GonkConsumerBaseJB.cpp
rename from widget/gonk/nativewindow/GonkConsumerBase.cpp
rename to widget/gonk/nativewindow/GonkConsumerBaseJB.cpp
diff --git a/widget/gonk/nativewindow/GonkConsumerBase.h b/widget/gonk/nativewindow/GonkConsumerBaseJB.h
rename from widget/gonk/nativewindow/GonkConsumerBase.h
rename to widget/gonk/nativewindow/GonkConsumerBaseJB.h

